docs/contributing.md

```diff
@@ -1,4 +1,25 @@
 # Contributing
 
-We follow the same guidelines used by testcontainers-java -- please [check them
-out](https://www.testcontainers.org/contributing/).
+* Star the project on [Github](https://github.com/testcontainers/testcontainers-go) and help spread the word :)
+* Join our [Slack group](http://slack.testcontainers.org)
+* [Post an issue](https://github.com/testcontainers/testcontainers-go/issues) if you find any bugs
+* Contribute improvements or fixes using a [Pull Request](https://github.com/testcontainers/testcontainers-go/pulls). If you're going to contribute, thank you! Please just be sure to:
+    * discuss with the authors on an issue ticket prior to doing anything big.
+    * follow the style, naming and structure conventions of the rest of the project.
+    * make commits atomic and easy to merge.
+    * when updating documentation, please see [our guidance for documentation contributions](contributing_docs.md).
+    * apply format running `go fmt`, or the shell script `./scripts/checks.sh`
+    * verify all tests are passing. Build and test the project with `make test-all` to do this.
+
+## Combining Dependabot PRs
+
+Since we generally get quite a few Dependabot PRs, we regularly combine them into single commits. 
+For this, we are using the [gh-combine-prs](https://github.com/rnorth/gh-combine-prs) extension for [GitHub CLI](https://cli.github.com/).
```

Comment:
Did you ever use the GH CLI plugin before in `tc-go`?

Reply:
No, and I was tempted to remove the entire section, as I copied it from tc-java, and we do not receive as much @dependabot PRs (https://github.com/testcontainers/testcontainers-go/pulls?q=is%3Apr+author%3Aapp%2Fdependabot+is%3Aclosed)

In fact, I think we should remove it

Reply:
Let's remove it if this does not reflect the reality 👍

Reply:
Done, thanks!